aioshelly/wsrpc.py

```diff
@@ -202,10 +210,16 @@ def connected(self) -> bool:
         """Return if we're currently connected."""
         return self._client is not None and not self._client.closed
 
-    async def call(self, method, params=None, timeout=10):
+    async def call(
+        self, method: str, params: dict[str, Any] | None = None, timeout: int = 10
+    ) -> Any:
```

Comment:
No need to return `Any` if we add raise on error
```suggestion
    ) -> dict[str, Any]:
```